tools/cldr-code/src/test/resources/org/unicode/cldr/unittest/data/common/dtd/ldml.dtd

```diff
@@ -2051,7 +2051,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 <!ATTLIST decimal references CDATA #IMPLIED >
```

Comment:
I'm wondering why this file exists; I've noticed that it causes errors when it drifts way from the regular ldml.dtd. If we keep it, we should have a test that it is identical (and maybe we do, and that's what alerted you to it?)

Reply:
In eclipse I see the following error with that file:

Description	Resource	Path	Location	Type
The element 'gmtUnknownFormat' has not been declared.	ldml.dtd	/cldr-code/src/test/resources/org/unicode/cldr/unittest/data/common/dtd	line 1723	DTD Problem

Reply:
I'm confused by that too. There are 3 ldml.dtd files:

1. common/dtd/ldml.dtd
2. tools/cldr-code/src/test/resources/org/unicode/cldr/unittest/data/common/dtd/ldml.dtd
3. tools/cldr-code/target/test-classes/org/unicode/cldr/unittest/data/common/dtd/ldml.dtd

I was accidentally editing the wrong one before I noticed that...

Reply:
In fact I did that again even after realizing there are 3 files; this PR still fails maybe for that reason. I'll make another (4th) commit shortly.

@srl295 do you have insight about how the ldml.dtd files relate to each other?